windows/micropython.vcxproj

```diff
@@ -75,10 +76,25 @@
   <PropertyGroup Label="UserMacros">
     <CustomPropsFile Condition="'$(CustomPropsFile)'==''">msvc/user.props</CustomPropsFile>
   </PropertyGroup>
-  <PropertyGroup />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <OutDir>$(PyOutDir)$(Configuration)\$(Platform)\</OutDir>
+  </PropertyGroup>
```

Comment:
Compiler/linker configuration should go in the corresponding props file, common.props in this case, in order not to repeat yourself for all possible configuration/platform combos.
In any case it's debatable whether the output directory should change: all other ports use the same principle, and I guess there's a good reason for it @pfalcon @dpgeorge ?